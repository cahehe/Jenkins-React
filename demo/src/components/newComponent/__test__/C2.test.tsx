import {render, screen, cleanup} from "@testing-library/react"
import renderer from 'react-test-renderer'
import C2 from "../C2"

it("should display hello world", () =>{
    render(<C2/>)
    const element = screen.getByTestId("1")
    expect(element).toBeInTheDocument
})

test('matchse snapshot', () => {    
    const tree = renderer.create(<C2/>).toJSON();
    expect(tree).toMatchSnapshot();
})