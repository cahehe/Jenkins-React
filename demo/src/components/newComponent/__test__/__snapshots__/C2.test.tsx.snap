// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matchse snapshot 1`] = `
<div
  data-testid="1"
>
  Hello there
</div>
`;
